DATA/production/qc-async/ft0.json

```diff
@@ -3,43 +3,40 @@
       "config": {
         "database": {
           "implementation": "CCDB",
-          "host": "qcdb.cern.ch:8083",
+          "host": "ali-qcdb.cern.ch:8083",
```

Comment:
This is irrelevant, right @knopers8 ?

Reply:
Right, it is overwritten by the central config.

Reply:
Yes, this is what I meant, also for the following comments.